web/controllers/caseload/user_controller.ex

```diff
@@ -22,6 +22,8 @@ defmodule Healthlocker.Caseload.UserController do
   end
 
   def show(conn, %{"id" => id}) do
-    show(conn, %{"id" => id, "section" => "details"})
+    user = Repo.get!(User, id)
```

Comment:
We need this query to scope whether the clinician actually has permission to view this `user_id`, are they part of their care team. Otherwise, I can just put any id in the URL.